### ISSUE TITLE
Something strange with proot and binary-bootstrap

### ISSUE THREAD

**Issue by cr6git** [voidlinux/void-packages#14167]
### System

* xuname:  `Void 4.16.7_1 x86_64 GenuineIntel uptodate rr`

* package:  `proot-5.1.0_5`

### Steps to reproduce the behavior

Very simple to demonstrate.
Using **xbps-uunshare** (the default method) there is no problem:
````
$ ./xbps-src binary-bootstrap

$ ./xbps-src pkg <any-package>

=> Reconfiguring bootstrap packages...
glibc-locales: configuring ...
Generating GNU libc locales...
  en_US.UTF-8... done.
glibc-locales: configured successfully.
ca-certificates: configuring ...
ca-certificates: configured successfully.
````

However, using **proot** it will throw an error:
````
$ ./xbps-src binary-bootstrap

$ ./xbps-src pkg <any-package>

=> Reconfiguring bootstrap packages...
glibc-locales: configuring ...
Generating GNU libc locales...
  en_US.UTF-8.../tmp/.xbps-script-hWyWQX: line 33:  4186 Segmentation fault      localedef --prefix=. -i $input -c -f $charset -A $LOCALE_ALIAS $locale
 done.
glibc-locales: configured successfully.
ca-certificates: configuring ...
ca-certificates: configured successfully.
````
:confused:

**Comment by cr6git** [comment]
This morning no more errors! :smile:

**Comment by cr6git** [comment]
it closes [#14167](https://github.com/voidlinux/void-packages/issues/14167) too.
